<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VaultKee</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="40,100" columnminimumwidth="0,0">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="objectViewLayout" stretch="10,4">
        <item>
         <widget class="QTableWidget" name="objectsViewer">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Comments</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>asd</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>asd</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>asd</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>asd</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>asd</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="objectViewer">
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="pathTreeWidget">
      <column>
       <property name="text">
        <string>Paths</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
